backend/main/services/matching/matcher.py

```diff
@@ -6,7 +6,7 @@
     cast,
 )
 
-import duckdb
+import duckdb  # type: ignore[import-untyped]
```

Comment:
This was copied from https://github.com/tuva-health/tuva_empi/pull/114 - to linting error in CI.